src/System.Text.Json/tests/Serialization/Array.ReadTests.cs

```diff
@@ -174,6 +174,28 @@ public static void ReadPrimitiveArray()
             Assert.Equal(0, i.Length);
         }
 
+        [ActiveIssue(38435)]
+        [Fact]
+        public static void ReadInitializedArrayTest()
```

Comment:
@steveharter @ahsonkhan this is the issue i mentioned

Reply:
It's probably worth updating the issue title then to reflect the actual issue:
https://github.com/dotnet/corefx/issues/38435 is currently titled as "Struct with an object not parsed successfully" which isn't what this test is verifying.